apps/gp2-frontend/src/events/Event.tsx

```diff
@@ -18,7 +19,14 @@ const Event: React.FC = () => {
           backHref={backHref}
           displayCalendar={false}
           eventOwner={<div>GP2 Team</div>}
-        />
+        >
+          {event.speakers.length && (
+            <>
+              <EventSpeakers speakers={event.speakers} />
+              <Divider />
```

Comment:
we can pass the `Divider` to inside `EventSpeaker`, the `div` container doesn't seem have any spacing bottom